### ISSUE TITLE
GitHub dependencies do not disappear from the lock file after they are removed

### ISSUE THREAD

**Issue by teo-tsirpanis** [fsprojects/Paket#3262]
### Description

I believe that the title is self-descriptive. Moreover, the entry in `paket.lock` stays the same even after the branch or the commit hash are explicitly changed in `paket.dependencies`.

### Reproduction steps

Please provide the steps required to reproduce the problem

1. Add a GitHub dependency.

2. `paket install`

3. Remove it.

4. `paket install`

### Expected behavior

`paket.lock` does not state the now-removed GitHub dependency.

### Actual behavior

The dependency still exists in `paket.lock`.

### Known workarounds

Manually edit `paket.lock`.

**Comment by forki** [comment]
Run update. It should work

Theodore Tsirpanis <notifications@github.com> schrieb am Mi., 20. Juni
2018, 15:32:

**Comment by teo-tsirpanis** [comment]
Indeed, `paket update` removes it. But shouldn't `paket install` do so as well?

**Comment by forki** [comment]
It should. But change detection is not implemented yet for github deps.
Help would be welcomed

Theodore Tsirpanis <notifications@github.com> schrieb am Mi., 20. Juni
2018, 15:38:
